worker/worker.go

```diff
@@ -26,14 +29,15 @@ type Worker struct {
 	features      map[string]string
 	remoteContent bool
 	rx            RxFunc
+	cancelRX      CancelRxFunc
```

Comment:
Go naming conventions tend to avoid capitalizations except in very commonly
accepted cases such as "messageID".

```suggestion
	cancelRx      CancelRxFunc
```